modules/consentManagement.js

```diff
@@ -99,7 +99,7 @@ function lookupIabConsent(cmpSuccess, cmpError, hookConfig) {
     if (success) {
       if (tcfData.eventStatus === 'tcloaded' || tcfData.eventStatus === 'useractioncomplete') {
         cmpSuccess(tcfData, hookConfig);
-      } else if (tcfData.eventStatus === 'cmpuishown' && tcfData.tcString.length > 0 && tcfData.purposeOneTreatment === true) {
+      } else if (tcfData.eventStatus === 'cmpuishown' && tcfData.tcString && tcfData.purposeOneTreatment === true) {
```

Comment:
in case `tcfData.tcString` is not defined

Reply:
I'm not sure about this change.

The original spec asked for this use-case that the `tcString` had to exist.  Further if there was a scenario where the tcString was undefined and we tried to process it, the module should reject anyways (see [here](https://github.com/prebid/Prebid.js/blob/master/modules/consentManagement.js#L308)).

Reply:
@jsnellbaker it may be true that the spec says it should always be a string, but that doesn't mean we shouldn't defend against bad data.  @robertrmartinez saw a console error during testing, so it must have reached this code.  But ultimately there's no negative impact because the function just exits anyway.   So not a big deal now.  But if the code changes at some point, it might be a problem.